### ISSUE TITLE
Cannot install with ESLint 4

### ISSUE THREAD

**Issue by MikeMcC399** [lo1tuma/eslint-plugin-mocha#355]
## Issue

The [README > Install and configure](https://github.com/lo1tuma/eslint-plugin-mocha/blob/main/README.md#install-and-configure) for `eslint-plugin-mocha@10.4.3` says:

> This plugin requires ESLint 4.0.0 or later.

however attempting to install with ESLint `4.0.0` fails with:

> npm ERR! Could not resolve dependency:
> npm ERR! peer eslint@">=7.0.0" from eslint-plugin-mocha@10.4.3

## Steps to reproduce

On Ubuntu `22.04.4` LTS with Node.js `20.12.2` LTS

```shell
mkdir eslint-plugin-mocha-test
cd eslint-plugin-mocha-test
git init
npm init -y
npm install eslint@4.0.0 eslint-plugin-mocha@10 -D
```

Note the error

> npm ERR! code ERESOLVE
> npm ERR! ERESOLVE unable to resolve dependency tree

## Logs

```text
$ npm install eslint@4 eslint-plugin-mocha@10 -D
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: eslint-plugin-mocha-test@1.0.0
npm ERR! Found: eslint@4.19.1
npm ERR! node_modules/eslint
npm ERR!   dev eslint@"4" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer eslint@">=7.0.0" from eslint-plugin-mocha@10.4.3
npm ERR! node_modules/eslint-plugin-mocha
npm ERR!   dev eslint-plugin-mocha@"10" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

## Suggestion

- PR https://github.com/lo1tuma/eslint-plugin-mocha/pull/247 in May 2020 updated `peerDependencies` from
  `eslint >=4.0.0` to
  `eslint >=7.0.0`
    so it would make sense to update the README accordingly.

Suggest to update the [README > Install and configure](https://github.com/lo1tuma/eslint-plugin-mocha/blob/main/README.md#install-and-configure) section to say:

> This plugin requires ESLint 7.0.0 or later.

**Comment by lo1tuma** [comment]
Good catch, thank you. Since the breaking change was announced via the release notes, I think we are good and just need to update the README, as you suggested.

**Comment by MikeMcC399** [comment]
@lo1tuma 

> Good catch, thank you. Since the breaking change was announced via the release notes, I think we are good and just need to update the README, as you suggested.

Yes, I see this under the release notes for [7.0.0](https://github.com/lo1tuma/eslint-plugin-mocha/releases/tag/7.0.0). Thanks for pointing that out.

**Comment by MikeMcC399** [comment]
@lo1tuma 

The updated README is not visible on https://www.npmjs.com/package/eslint-plugin-mocha.

If you are removing support for ESLint 8 and below, you might want to wait until ESLint 8 becomes generally end-of-life on Oct 5, 2024 (see https://eslint.org/version-support/) and cut a new major breaking version 11.x .

**Comment by lo1tuma** [comment]
Yes, I’m planning a few more breaking changes, for the next major release. I doubt that I will finish that before Oct 5.

**Comment by MikeMcC399** [comment]
@lo1tuma 

> Yes, I’m planning a few more breaking changes, for the next major release. I doubt that I will finish that before Oct 5.

One alternative is to first deprecate the earlier ESLint versions before dropping support. That wouldn't need a breaking major release. Just a thought for you 🙂 !

**Comment by MikeMcC399** [comment]
@lo1tuma 

https://www.npmjs.com/package/eslint-plugin-mocha?activeTab=readme stills shows the old text:

![Image](https://github.com/user-attachments/assets/b101d394-3264-41c5-8d9b-af7ced3154bf)
